sdk/azidentity/version.go

```diff
@@ -10,5 +10,5 @@ const (
 	UserAgent = "azidentity/" + Version
 
 	// Version is the semantic version (see http://semver.org) of this module.
-	Version = "v0.6.1"
+	Version = "v0.9.1"
```

Comment:
Assuming you're leaving this as 0.9.1 for the next release, right?

Reply:
Yep!